lib/chef/knife/core/subcommand_loader.rb

```diff
@@ -28,8 +28,8 @@ class SubcommandLoader
       attr_reader :chef_config_dir
       attr_reader :env
 
-      def initialize(chef_config_dir, env=ENV)
```

Comment:
Do folks subclass this a lot?  I can't see @env being used anywhere except to get HOME during the plugin search.  Even then, it's used inconsistently because ENV is used when looking for the manifest instead.